cmd/wstail/tail.go

```diff
@@ -42,9 +42,15 @@ func writer(ws *websocket.Conn, filter filter) {
 	log.Infof("active filter: %v (%v)", filter, matcher)
 	defer ws.Close()
 
-	tailCmd := exec.Command("tail", fmt.Sprintf("-n%v", tailHistory), "-f", filePath)
+	var err error
+	var history io.ReadCloser
+	tailCmd := exec.Command("tail", "-f", filePath)
 	commands := []*exec.Cmd{tailCmd}
 	if matcher != "" {
+		history, err = snapshot(matcher, filePath, tailHistory)
+		if err != nil {
+			log.Errorf("failed to obtain %v lines of history for %v: %v", tailHistory, matcher, trace.DebugReport(err))
```

Comment:
This is probably generally a warn (i.e. not enough history) and in the case that it's an actual error, should we abort here?